### PR TITLE
Wrappers: add support for rr

### PR DESCRIPTION
**Pull request by clebergnu**
This example wrapper script runs binaries inside rr (record), which
saves the execution environment, allowing to deterministically debug
possible failures later.

The usage instructions to run tests with this wrapper are no different
than other wrappers.  The effect of the wrapper, though, deserves notice:
it creates an "rr" directory inside the test results, which rr uses
to save its data files.

Replays should be executed like this:

 $ rr replay <job-results>/test-results/<test>/rr/<binary-number>

Where <binary-number> is the "Nth" execution of "binary" by your
test.

Signed-off-by: Cleber Rosa <crosa@redhat.com>

### CODE REVIEW

File: examples/wrappers/rr.sh [hunk 1]
```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+#
+# Record deterministic execution using rr
```
Comment (adereis) [inline examples/wrappers/rr.sh]: http://rr-project.org/

Comment (clebergnu) [inline examples/wrappers/rr.sh]: ACK.

File: examples/wrappers/rr.sh [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+#
+
+export _RR_TRACE_DIR=$AVOCADO_TEST_LOGDIR/rr
+mkdir -p $_RR_TRACE_DIR
+exec rr record "$@"
```
Comment (ldoktor) [inline examples/wrappers/rr.sh]: Yep, when I add `-n` it works fine. Maybe one note the test results is standardized and should not contain anything except the files mentioned in the documentation. How about putting the tracedir into `$AVOCADO_TEST_LOGDIR/data`?

Comment (clebergnu) [inline examples/wrappers/rr.sh]: Yep, good idea (that is actually `$AVOCADO_TEST_OUTPUTDIR`).

**Comment by clebergnu** [comment]
Closing in favor of #1353

**Pull request closed without merging**